src/utils/art-concepts.ts

```diff
@@ -0,0 +1,7 @@
+export let artConcepts = new Map([
```

Comment:
@kajambiya Do you need a new file for this under utils? This is scope specific also. I would add this map object to the [art-therapy-tab.component.tsx](https://github.com/UCSF-IGHS/openmrs-esm-ohri/pull/872/files#diff-7498cdb3de2173857d6b6eb1958955a38fa219d2f737f582519645291768cf78) file.